apps/datadog-api/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Datadog API",
+  "description": "A collection of blocks representing low-level API calls for Datadog API.",
```

Comment:
See slack - in the end this also includes high-level blocks like the subscription block.